app/views/handlers/list_action.py

```diff
@@ -33,7 +33,7 @@ def is_location_valid(self):
         return True
 
     def get_previous_location_url(self):
-        if self._return_to_summary:
+        if self._return_to == "summary":
```

Comment:
I was expecting to have to deal return_to="final_summary here" however whether it is by design or a bug, you can't have list items appear on the final summary. https://github.com/ONSdigital/eq-questionnaire-runner/blob/master/app/views/contexts/questionnaire_summary_context.py#L33

Reply:
By design for Census.

Reply:
Thanks Mebin